lib/workload/stateless/filemanager/.sqlx/query-6515aa29536fb79eb777151f8650c8eec41a9bd4bcc72a06d36a5d0eed482115.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
FYI, these `query-*.json` files are required for filemanager to compile without an active postgres database running, which means that any `cdk synth` doesn't need to know about the filemanager's docker compose configuration.